package.json

```diff
@@ -19,18 +19,20 @@
     "!test.js"
   ],
   "dependencies": {
-    "file-type": "<=16.5.1",
+    "file-type": "^18.2.1",
```

Comment:
What was "the warning" you were getting about `file-type`? If it's a security vuln, you can [use 16.5.4 instead](https://github.com/sindresorhus/file-type/releases/tag/v16.5.4).

Reply:
Likely that will fix it yes

https://socket.dev/npm/package/file-type/overview/16.5.1

Reply:
Yeah gone here
https://socket.dev/npm/package/file-type/overview/16.5.4

Reply:
If I make that change tmrw is it fine to mergee
Ie just jump to latest 16?

Reply:
> If I make that change tmrw is it fine to mergee
> Ie just jump to latest 16?

If you

- Update file-type to latest 16
- Use `require` not `import`
- Tests pass

Then yes you can merge without my review :)

Reply:
yeah reverted the ESM stuff, done as you're saying. Waiting for CI to pass then will merge-publish.

Thanks for the attention